ogr/ogrsf_frmts/gpkg/ogrgeopackagetablelayer.cpp

```diff
@@ -2039,6 +2039,56 @@ OGRErr OGRGeoPackageTableLayer::ISetFeature( OGRFeature *poFeature )
     return eErr;
 }
 
+/************************************************************************/
+/*                          FeatureIDExists()                           */
+/************************************************************************/
+
+bool OGRGeoPackageTableLayer::FeatureIDExists( GIntBig nFID )
+{
+   CPLString soSQL;
+   soSQL.Printf("SELECT %s FROM \"%s\" m "
+                "WHERE \"%s\" = " CPL_FRMT_GIB,
+                m_soColumns.c_str(),
```

Comment:
could be simplified as:
```suggestion
   soSQL.Printf("SELECT 1 FROM \"%s\" "
                "WHERE \"%s\" = " CPL_FRMT_GIB,
```